src/Value/Color.php

```diff
@@ -72,23 +72,48 @@ public static function parse(ParserState $oParserState, bool $bIgnoreCase = fals
             $oParserState->consumeWhiteSpace();
             $oParserState->consume('(');
 
+            // CSS Color Module Level 4 says that `rgb` and `rgba` are now aliases; likewise `hsl` and `hsla`.
+            // So, attempt to parse with the `a`, and allow for it not being there.
+            switch ($sColorMode) {
+                case 'rgb':
+                case 'hsl':
+                    $colorModeForParsing = $sColorMode . 'a';
```

Comment:
I think it would reduce the cognitive complexity if we used string literals for `$colorModeForParsing` in the different cases (except for the `default` case)  instead of using `$sColorMode`.

Reply:
Done.